src/services/codefixes/helpers.ts

```diff
@@ -107,7 +107,18 @@ namespace ts.codefix {
         return nodes && createNodeArray(nodes.map(getSynthesizedDeepClone));
     }
 
-    export function createMethodFromCallExpression({ typeArguments, arguments: args }: CallExpression, methodName: string, inJs: boolean, makeStatic: boolean): MethodDeclaration {
+    export function createMethodFromCallExpression(context: CodeFixContextBase, { typeArguments, arguments: args }: CallExpression, methodName: string, inJs: boolean, makeStatic: boolean): MethodDeclaration {
+        const checker = context.program.getTypeChecker();
+        const types = map(args,
+            (arg) => {

```

Comment:
I'd delegate this to a linter, but looks like it's not enabled here, so but no parens around single parameters in arrow functions